deployer/templates/hawkular-cassandra-node-pv.yaml

```diff
@@ -77,7 +77,7 @@ objects:
             containerPort: 7001
           command:
           - "/opt/apache-cassandra/bin/cassandra-docker.sh"
-          - "--seed_provider_classname=org.hawkular.openshift.cassandra.OpenshiftSeedProvider"
+          - "--seed_provider_classname=org.apache.cassandra.locator.SimpleSeedProvider"
```

Comment:
In the previous version we were using a custom seed provider. This meant for the one running in OpenShift we would use the custom seed provider, while if you wanted to run just the plain docker image (without OpenShift) you could use the default one. If you wanted to use a custom seed provider you could also extend our base image and change the command option to point to the one you want to use instead.

We can probably remove the option in the configuration and just have it use the default, but I don't think we would want to remove the functionality.